umm-spec-lib/src/cmr/umm_spec/test/iso_shared.clj

```diff
@@ -96,6 +96,9 @@
 (defn expected-doi
   "Returns the expected DOI."
   [doi]
-  (let [updated-doi (util/remove-nil-keys (dissoc doi :MissingReason :Explanation))]
-    (when (seq updated-doi)
-      (cmn/map->DoiType updated-doi))))
+  (let [explanation (when (seq (:Explanation doi))
```

Comment:
Similarly shouldn't need to call `seq` here.

Reply:
Same request for clarity as above ...